cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -102,32 +88,24 @@ func CreateKubeDNSRBACClusterRole(clientset *clientset.Clientset) error {
 	return nil
 }
 
-// CreateKubeProxyClusterRoleBinding creates the necessary ClusterRole for kube-dns
+// CreateKubeProxyClusterRoleBinding grants the system:node-proxier role to the nodes group,
+// since kubelet credentials are used to run the kube-proxy
+// TODO: give the kube-proxy its own credential and stop requiring this
 func CreateKubeProxyClusterRoleBinding(clientset *clientset.Clientset) error {
-	systemKubeProxySubject := rbac.Subject{
-		Kind:      "User",
-		Name:      "system:kube-proxy",
-		Namespace: api.NamespaceSystem,
-	}
-
-	systemNodesSubject := rbac.Subject{
-		Kind:      "Group",
-		Name:      "system:nodes",
-		Namespace: api.NamespaceSystem,
-	}
-
 	clusterRoleBinding := rbac.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "system:node-proxier",
+			Name: "kubeadm:node-proxier",
```

Comment:
Is there any specific reason we prefix the [cluster]rolebindings with `kubeadm` and not system?

I see the ClusterRoles are prefixed system because those roles are automatically bootstrapped by the apiserver, but why can't we use `system:` for the bindings as well?

I'm thinking about portability here...